package.json

```diff
@@ -26,7 +26,8 @@
   "devDependencies": {
     "all-contributors-cli": "^6.26.1",
     "concurrently": "^8.2.1",
-    "prettier": "3.1.0",
-    "husky": "^8.0.0"
+    "eslint-plugin-import": "^2.29.1",
```

Comment:
you seem to have installed it globally rather than in the `client` dir

Reply:
My bad, Fixed it now.